From 8bc3f8e449f2207b60d971b637ca0821e9606070 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Fri, 22 Jun 2018 15:34:18 +0800
Subject: [PATCH] arm64: dts: rockchip: set logo to center display for
 rk3399-linux

set uboot and kernel logo center display for linux product.

Change-Id: I481432b2719598b5d776eb39c4809c05dfb646b0
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
index 20c45193664d..2851cd529e04 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
@@ -127,7 +127,7 @@
 			status = "disabled";
 			logo,uboot = "logo.bmp";
 			logo,kernel = "logo_kernel.bmp";
-			logo,mode = "fullscreen";
+			logo,mode = "center";
 			charge_logo,mode = "center";
 			connect = <&vopl_out_hdmi>;
 		};
@@ -136,7 +136,7 @@
 			status = "disabled";
 			logo,uboot = "logo.bmp";
 			logo,kernel = "logo_kernel.bmp";
-			logo,mode = "fullscreen";
+			logo,mode = "center";
 			charge_logo,mode = "center";
 			connect = <&vopb_out_dsi>;
 		};
@@ -145,7 +145,7 @@
 			status = "disabled";
 			logo,uboot = "logo.bmp";
 			logo,kernel = "logo_kernel.bmp";
-			logo,mode = "fullscreen";
+			logo,mode = "center";
 			charge_logo,mode = "center";
 			connect = <&vopb_out_edp>;
 		};
-- 
2.35.3

